src/reader/pi.rs

```diff
@@ -57,10 +57,9 @@ impl PiParser {
         for i in memchr::memchr_iter(b'>', bytes) {
             match i {
                 // +1 for `>` which should be included in event
```

Comment:
I believe this comment should be removed?

Reply:
Yes